python/ngraph/ops.py

```diff
@@ -78,6 +78,22 @@ def elu(data, alpha, name=None):  # type: (NodeInput, NodeInput, str) -> Node
     return Elu(as_node(data), as_node(alpha))
 
 
+@nameable_op
+def grn(data, bias, name=None):  # type: (Node, float, str) -> Node
+    r"""Perform Global Response Normalization with L2 norm (across channels only).
+
+    Computes GRN operation on channels for input blob:
```

Comment:
I don't think we're using the word `blob` in our documentation so far. Let's keep it consistent.